embed.html

```diff
@@ -0,0 +1,35 @@
+        Bokeh.load_models(all_models);
+        var plots = [{'modeltype': 'Plot', 'elementid': '#8a571eab-100a-403d-b772-22bfc498d44f', 'modelid': '4fdb5b27-4f37-4992-b82a-53a162a0e363'}, {'modeltype': 'Plot', 'elementid': '#6b798abf-c715-49d2-ae4d-bee9b2831e66', 'modelid': 'd166fd74-d6fa-403c-bd77-83894728579d'}, {'modeltype': 'Plot', 'elementid': '#2ce8d176-a29d-41d1-b02c-fe34eb342ab5', 'modelid': 'a2700b41-a966-44e0-90fa-aa356fb996c4'}]
+        for (idx in plots) {
+        	var plot = plots[idx]
+        	var model = Bokeh.Collections(plot.modeltype).get(plot.modelid);
+        	Bokeh.logger.info('Realizing plot:')
+        	Bokeh.logger.info(' - modeltype: ' + plot.modeltype);
+        	Bokeh.logger.info(' - modelid: ' + plot.modelid);
+        	Bokeh.logger.info(' - elementid: ' + plot.elementid);
+        	var view = new model.default_view({
+        		model: model,
+        		el: plot.elementid
+        	});
+        	Bokeh.index[plot.modelid] = view;
+        }
+    });
+</script>
+    </head>
+    <body>
+    <div class="plotdiv" id="8a571eab-100a-403d-b772-22bfc498d44f"></div><div class="plotdiv" id="6b798abf-c715-49d2-ae4d-bee9b2831e66"></div><div class="plotdiv" id="2ce8d176-a29d-41d1-b02c-fe34eb342ab5"></div>
+    </body>
+</html>
+    
```

Comment:
Weird I thought we had an Git ignore policy to ignore `.html` files under `examples`. Anyway, yah I agree we just want to check in the source `embed.py` and let users run it for themselves to generate the output file.